cmd/find-main.go

```diff
@@ -123,37 +123,34 @@ FORMAT
       {url} --> Substitutes to a shareable URL of the path.
 
 EXAMPLES:
-   01. Find all files named foo from all buckets.
-       $ {{.HelpName}} s3 --name "file"
+   01. Find all "foo.jpg" from all buckets from "s3" account
+       $ {{.HelpName}} s3 --name "foo.jpg"
 
-   02. Find all text files from mybucket.
+   02. Find all objects ending with ".txt" extension under "mybucket"
        $ {{.HelpName}} s3/mybucket --name "*.txt"
 
-   03. Print only the object names without the directory component under this bucket.
-       $ {{.HelpName}} s3/bucket --name "*" -print {base}
+   03. Print only the object names without the directory component under "mybucket".
+       $ {{.HelpName}} s3/mybucket --name "*" -print {base}
 
-   04. Copy all jpeg files from AWS S3 "s3/photos" bucket to minio "play/photos" bucket.
-       $ {{.HelpName}} s3/photos --name "*.jpg" --exec "mc cp {} play/photos"
-
-   05. Find all jpeg images from any folder prefixed with "album".
+   04. Find all jpg images from any folder prefixed with "album".
        $ {{.HelpName}} s3/photos --name "*.jpg" --path "*/album*/*"
 
-   06. Find all files with extensions jpeg, png, and gif using regex.
+   05. Find all objects with extensions jpg, png, and gif using regex.
        $ {{.HelpName}} s3/photos --regex "(?i)\.(jpg|png|gif)$"
 
-   07. Find all jpeg images from "s3/bucket" and copy to minio "play/bucket" *continuously*.
+   06. Find all jpg images from "s3/bucket" and copy to minio "play/bucket" *continuously*.
        $ {{.HelpName}} s3/bucket --name "*.jpg" --watch --exec "mc cp {} play/bucket"
 
-   08. Generate self expiring urls (7 days), for all objects between 64 MB, and 1 GB in size.
+   07. Generate public URLs valid for 7 days, for all objects between 64 MB, and 1 GB in size.
        $ {{.HelpName}} s3 --larger 64MB --smaller 1GB --print {url}
 
-   09. Find all files under the s3 bucket which were created within a week.
+   08. Find all objects created in last week.
        $ {{.HelpName}} s3/bucket --newer 1w
 
-   10. Find all files which were created more than 6 months ago ignoring files ending in jpg.
+   09. Find all objects which were created more than 6 months ago, additionally ignore all jpg objects.
```

Comment:
`Find all objects which were created more than 6 months ago, additionally ignore all jpg objects.`
 =>
`Find all objects created more than 6 months ago, and exclude the ones with ".jpg" extension under "s3".`